sway/commands.c

```diff
@@ -1214,37 +1247,54 @@ enum cmd_status handle_command(char *_exec) {
 				strip_quotes(argv[1]);
 			}
 			struct cmd_handler *handler = find_handler(argv[0]);
-			enum cmd_status res = CMD_INVALID;
-			if (!handler
-					|| (res = handler->handle(argc-1, argv+1)) != CMD_SUCCESS) {
-				sway_log(L_ERROR, "Command '%s' failed", cmd);
+			if (!handler) {
+				if (results) {
+					free_cmd_results(results);
+				}
+				results = cmd_results_new(CMD_INVALID, strdup(cmd), "Unknown/invalid command");
```

Comment:
`cmd` is freed later in this method, so I need to duplicate it. However, in the struct that string is never free-ed since it's supposed to be a `const char*` ... What's the right thing to do here?